latex/expressingParallelism.tex

```diff
@@ -2249,6 +2249,66 @@ \subsubsection{Defining kernels using OpenCL C kernel objects}
 \codeinline{cl_kernel} is no longer needed in the calling context.
 
 
+%*******************************************************************************
+% OpenCL Kernel Conventions and SYCL
+%*******************************************************************************
+\subsection{OpenCL Kernel Conventions and SYCL}
+\label{sec:opencl_kern_conventions_and_sycl}
+
+OpenCL and SYCL use opposite conventions for the unit stride dimension.  SYCL
+aligns with C++ conventions, which is important to understand from a performance
+perspective when porting code to SYCL.
+
+When specifying a \codeinline{range} as the global or local size
+in a \codeinline{parallel_for} that invokes an OpenCL interop kernel (through
+\codeinline{cl_kernel} interop or \codeinline{compile_with_source}/
+\codeinline{build_with_source}),
+the highest dimension of the range in SYCL will map to the
+lowest dimension within the OpenCL kernel.  That statement applies to both
+an underlying enqueue operation such as \codeinline{clEnqueueNDRangeKernel}
+in OpenCL, and also ID and size queries within the OpenCL kernel.
+For example, a global range specified in SYCL as:
```

Comment:
I think we should specify the 2-dimensional case here as well to clarify that only two dimensions are used otherwise there can be some ambiguity in how they are passed to `clEnqueueNDRangeKernel` after the flip.

I.e. `{r0, r1}` -> `{r1, r0}`, rather than `{1, r1, r0}`.

Reply:
This is covered by the new addition, thanks.